invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -251,6 +251,21 @@ def dump(self):
         return self._vocabularies
 
 
+def load_custom_fields(conf_ui, conf_backend):
+    """Load custom fields configuration."""
+    from invenio_vocabularies.custom_fields import VocabularyCF
+
+    for section_cfg in conf_ui:
+        vocabulary_fields = []
+        fields = section_cfg["fields"]
+        for field in fields:
+            field_type = conf_backend.get(field["field"])
+            if isinstance(field_type, VocabularyCF):
```

Comment:
we now have a `realtion_cls` attr in the field obj. so we can check if None or not. is implicit but avoid cast check

Reply:
https://github.com/inveniosoftware/invenio-records-resources/issues/358